erdkotlin/src/main/java/com/elrond/erdkotlin/data/transaction/TransactionRepositoryImpl.kt

```diff
@@ -24,7 +24,7 @@ internal class TransactionRepositoryImpl(
 
     override fun getTransactions(address: Address): List<TransactionOnNetwork> {
         val response = elrondProxy.getAddressTransactions(address)
-        return requireNotNull(response.data).transactions.map { it.toDomain() }
+        return response.data?.transactions?.map { it.toDomain() } ?: emptyList()
```

Comment:
If we are retrieving the transactions from an API call, we should signal errors properly:
- throw Exception
or
- Error result type, but this will require a method signature change

Reply:
This change handle the case where an address never made any transaction.
An empty list fits well since there are exactly 0 transactions.
Also, the API is answering with "success"
I don't think we should raise an exception here.
What do you think ?

Reply:
Yes, empty list in that case is correct. 
Does `elrondProxy.getAddressTransactions(address)` throw errors in case the API call fails?
e.g. 500 Internal Server Error?

Reply:
Yes, it does throw an exception in that case.

Reply:
Ok, thanks for your answers!